bazel/defs.bzl

```
@@ -204,15 +204,16 @@ def rust_ic_test(env = {}, data = [], **kwargs):
         **kwargs
     )
 
-def rust_bench(name, env = {}, data = [], pin_cpu = False, with_test = False, **kwargs):
+def rust_bench(name, env = {}, data = [], pin_cpu = False, test_name = None, test_timeout = None, **kwargs):
     """A rule for defining a rust benchmark.
 
     Args:
       name: the name of the executable target.
       env: additional environment variables to pass to the benchmark binary.
       data: data dependencies required to run the benchmark.
       pin_cpu: pins the benchmark process to a single CPU if set `True`.
-      with_test: generates name + '_test' target to test that the benchmark work.
+      test_name: generates test with name 'test_name' to test that the benchmark work.
+      test_timeout: timeout to apply when generating a test (default: `moderate`).
```

```suggestion
      test_timeout: timeout to apply in the generated test (default: `moderate`).
```
it first sounded like it's a bazel for the bazel action to produce the test target